MailCore/Utils/Matomo+Extension.swift

```diff
@@ -119,4 +119,24 @@ public extension MatomoUtils {
             track(eventWithCategory: .userInfo, action: .data, name: "multipleMessagesInThread", value: messagesCount)
         }
     }
+
+    func trackAction(action: Action, origin: ActionOrigin, numberOfItems: Int, isMultipleSelection: Bool) {
```

Comment:
Maybe a name more explicit would be a good idea? Something like `trackBottomSheetAction`?